Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml

```diff
@@ -515,21 +474,21 @@
 #      prob: 0.01
 #    - id: MobLuminousEntity
 #      prob: 0.01
-#
-#- type: entity
-#  name: Friendly Xeno spawner
-#  id: friendlyxenoSpawner
-#  parent: MarkerBase
-#  components:
-#    - type: Sprite
-#      layers:
-#        - state: green
-#        - sprite: Mobs/Aliens/Xenos/drone.rsi
-#          state: sleeping
-#    - type: RandomSpawner
-#      prototypes:
-#        - MobXenoNeutralRouny
-#        - MobXenoNeutralPraetorian
-#        - MobXenoNeutralDrone
-#        - MobXenoNeutralRavager
-#      chance: 1
+
+- type: entity
+  name: Friendly Xeno spawner
+  id: friendlyxenoSpawner
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: green
+        - sprite: Mobs/Aliens/Xenos/drone.rsi
+          state: sleeping
+    - type: RandomSpawner
+      prototypes:
+        - MobXenoNeutralRouny
+        - MobXenoNeutralPraetorian
+        - MobXenoNeutralDrone
+        - MobXenoNeutralRavager
+      chance: 1
```

Comment:
Not sure, but this probably belongs in a file with other mob spawners, like renault and other pets. This should be moved.